arelle/plugin/inlineXbrlDocumentSet.py

```diff
@@ -250,10 +250,10 @@ def createFacts(facts, parent):
                     text = None
                 elif ( not(modelConcept.baseXsdType == "token" and modelConcept.isEnumeration)
                        and fact.xValid ):
```

Comment:
Discussed offline. Check for xValid and remove the `INVALIDixVALUE` handling in the if branch:
```python
                elif ( not(modelConcept.baseXsdType == "token" and modelConcept.isEnumeration)
                       and fact.xValid >= VALID ):
                    text = fact.xValue
                # may need a special case for QNames (especially if prefixes defined below root)
                else:
                    text = fact.rawValue if fact.textValue == INVALIDixVALUE else fact.textValue
```